public/assets/css/style.css

```diff
@@ -3736,10 +3738,121 @@ hr {
 }
 .footer-social a i {
   display: inline-block;
+} */
+
+
+/* added by ePSA-eJya */}
+
+.footer-social ul {
+	display: flex;
+	margin: 0;
+	padding: 0;
 }
-.footer-bg {
-  background: #4255a4;
+
+.footer-social ul li {
+	list-style: none;
+}
+
+.footer-social ul li a {
+	position: relative;
+	width: 40px;
+	height: 40px;
+  font-size: 14px;
+	background-color: rgba(128, 136, 255, 0.1);
+	text-align: center;
+	line-height: 40px;
+	border-radius: 50%;
+	overflow: hidden;
+	z-index: 1;
+
+	color: #fff;
+	text-decoration: none;
+	font-size: 20px;
+	flex-basis: 35%;
+	margin-right: 8px;
+	display: inline-block;
+	transition: transform 0.5s;
```

Comment:
if you have include footer css in a file ? why here to?

Reply:
ya i will remove those